### ISSUE TITLE
Problems creating confluence page: Error parsing xhtml

### ISSUE THREAD

**Issue by paulwellnerbou** [gscheibel/asciidoctor-confluence#1]
I get this error message:

```
An error occurred, the page has not been created because:
Error parsing xhtml
```

The confluence (5.7.5) is behind an HTTPS URL. Should it work with HTTPS? Is this error message coming from confluence or from parsing the asciidoc result?

Is there a way to make this a bit more verbose?

**Comment by jaredmorgs** [comment]
Yep, I'm getting this issue as well. And my confluence host is behind HTTPS as well.

**Comment by kevinquillen** [comment]
Same here. No further error is given.

**Comment by rdmueller** [comment]
for me, this error message sound like it is coming from confluence.
You could check if you get the same error message with this groovy script: https://github.com/rdmueller/asciidoc2confluence

**Comment by kevinquillen** [comment]
For me, I had to convert ampersands to &amp; in URLs like links or img src values. Once I did that, I stopped getting this error (from Confluence).

**Comment by arjunshukla** [comment]
Use body.storage.value if you are fetching from confluence to post to another confluence.
Or refer this document: https://confluence.atlassian.com/doc/confluence-storage-format-790796544.html

**Comment by gbougeard** [comment]
I have the same issue. It looks like it does not understand some syntaxes like : 

```
`service` ::
 _(mandatory)_ ::
 name of the service
```

**Comment by mojavelinux** [comment]
If this problem still exists, it should be filed in the upstream repository.

See https://github.com/asciidoctor/asciidoctor-confluence

**Comment by gbougeard** [comment]
@mojavelinux so do you think this is a duplicate of https://github.com/asciidoctor/asciidoctor-confluence/issues/7 or shall we fill a new one?

**Comment by mojavelinux** [comment]
Yes, that appears to be the correct upstream issue. If a change was made, it would need to go into that repository, which is why I mention it.

**Comment by ghost** [comment]
If solution from  asciidoctor/asciidoctor-confluence#7 does not work, try with `-- -b xhtml5 -S safe -s`.

**Comment by rdmueller** [comment]
@maniedzi If you need to publish asciidoc to confluence, you might want to try the `publishToConfluence` feature of docToolchain. It is under active development. (I am one of the main committers)

another project capable of doing this is https://github.com/confluence-publisher/confluence-publisher
